app/definition/ApiDefinitionFactory.scala

```diff
@@ -17,7 +17,7 @@
 package definition
```

Comment:
General comment: Instead of deleting all the v4 OAS files, have we considered deleting the `v4/application.yaml` only? I think this is the only file that the platform checks directly, so if it's not there none of v4 will show on DevHub. And we'll still have (almost) all the v4 files ready to go when we need them :)